Cards/MadGraph5_aMCatNLO/SingleT/TWminusto2L2Nu-5FS-DR1_TuneCP5_13p6TeV_amcatnlo-pythia8/TWminusto2L2Nu-5FS-DR1_TuneCP5_13p6TeV_amcatnlo-pythia8_proc_card.dat

```diff
@@ -0,0 +1,7 @@
+import model loop_sm-no_b_mass
```

Comment:
let's change this to ckm version of the model (ttW/ttZ was too heavy but i think this is light enough to do ckm)

import model loop_sm-ckm_no_b_mass